apps/nextjs-website/src/components/Video.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+import { Video as VideoRC } from '@react-components/components';
+import { VideoImageProps } from '@react-components/types';
+import { VideoSection } from '@/lib/fetch/types/PageSection';
+
+const makeVideoProps = ({
+  title,
+  subtitle,
+  src,
+  fallback,
+  playButtonLabel,
+  isCentered,
+  ...rest
+}: VideoSection): VideoImageProps => ({
+  ...rest,
+  title,
+  subtitle,
+  ...(src.data && {
+    src: {
+      url: src.data.attributes.url,
+      alternativeText: src.data.attributes.alternativeText || null,
+      mime: src.data.attributes.mime,
+    },
+  }),
+  fallback,
+  playButtonLabel,
+  isCentered,
+});
```

Comment:
```suggestion
const makeVideoProps = ({
  src,
  ...rest
}: VideoSection): VideoImageProps => ({
  ...rest,
  ...(src.data && {
    src: {
      url: src.data.attributes.url,
      alternativeText: src.data.attributes.alternativeText || null,
      mime: src.data.attributes.mime,
    },
  }),
});
```
Remove unchanged data.